src/content/configuration/node.md

```diff
@@ -16,7 +16,7 @@ This feature is provided by webpack's internal [`NodeStuffPlugin`](https://githu
 
 ## `node`
 
-`object`
+`boolean | object`
```

Comment:
For enums we specify values, you cannot set it to `true` as schema wont let it go. So only mark false.

```suggestion
`bool: false | object`
```